### PR TITLE
Remove XFAIL from test that was failing after the llvm bump

### PR DESCRIPTION
**Pull request by flemairen6**

### CODE REVIEW

File: test/Conversion/TosaToXTenNN/quantization.mlir [hunk 1]
```diff
@@ -340,17 +337,16 @@ module attributes {} {
...
 // CHECK:             %[[VAL_2:.*]] = xten_nn.dequantize(%[[VAL_1]] : tensor<1x3x4x4xsi8>) {shift = -7 : si32} -> tensor<1x3x4x4xf32>
 // CHECK:             return %[[VAL_2]] : tensor<1x3x4x4xf32>
 // CHECK:           }
- func.func @fold_after_sort_on_mul() -> tensor<1x3x4x4xf32> {
+ func.func @fold_after_sort_on_mul(%arg0: tensor<1x3x4x4xf32>) -> tensor<1x3x4x4xf32> {
     %0 = "tosa.const"() {value = dense<1.280000e+02> : tensor<1x1x1x1xf32>} : () -> tensor<1x1x1x1xf32> 
     %1 = "tosa.const"() {value = dense<7.812500e-03> : tensor<1x1x1x1xf32>} : () -> tensor<1x1x1x1xf32> 
-    %2 = "tosa.const"() {value = dense<2.000000e-02> : tensor<1x3x4x4xf32>} : () -> tensor<1x3x4x4xf32> 
-    %3 = "tosa.mul"(%0, %2) {shift = 0 : i32} : (tensor<1x1x1x1xf32>, tensor<1x3x4x4xf32>) -> tensor<1x3x4x4xf32> 
+    %3 = "tosa.mul"(%0, %arg0) {shift = 0 : i32} : (tensor<1x1x1x1xf32>, tensor<1x3x4x4xf32>) -> tensor<1x3x4x4xf32>
```
Comment (cmcgirr-amd) [inline test/Conversion/TosaToXTenNN/quantization.mlir]: This changes the test case, does it not work using elided on that constant? We want to make sure we can find quantization of constant tensors like weights and biases.

Comment (flemairen6) [reply test/Conversion/TosaToXTenNN/quantization.mlir]: I looked for it, but is there such an attribute or a way to force a tosa.const to be elided?

Comment (cmcgirr-amd) [reply test/Conversion/TosaToXTenNN/quantization.mlir]: Does `value = dense_resource<__elided__>` work?

Comment (flemairen6) [reply test/Conversion/TosaToXTenNN/quantization.mlir]: It works, thanks for that then

**Review by cmcgirr-amd — APPROVED** [review]
Looks good!

**Pull request merged**